.mailmap

```diff
@@ -1537,3 +1537,4 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Øyvind Jensen <jensen.oyvind@gmail.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
+Jay Patankar <patankarjays@gmail.com>
```

Comment:
If you run the `bin/mailmap_check.py` script then it will reorder the lines in this file.